docs/release_notes/0.45.0.md

```diff
@@ -0,0 +1,21 @@
+# Release 0.45.0
+
+## Features
+
+- Use the AutoScalingGroup for getting & scaling nodegroups (#3543)
+- Make EKS 1.19 the default (#3526)
+- Support passing releaseVersion for Managed Nodegroups (#3544)
```

Comment:
```suggestion
- Support multiple AMI release versions in managed node groups
```

Reply:
is that accurate?

Reply:
Yeah, it doesn't seem accurate to me either but I was trying to use the title Mike used in the issue: https://github.com/weaveworks/eksctl/issues/3506

Reply:
meh i think i like your PR title better. i feel it is less likely to confuse people

Reply:
I think this is better:
```
- Support specifying an AMI release versions for managed node groups (#3544)
```

Reply:
oh yeh that's perfect